app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -1521,11 +1521,11 @@ private void setAndCalculatePokeSpamText(IVScanResult ivScanResult) {
                     pokeSpamCalculator.getTotalEvolvable(), pokeSpamCalculator.getEvolveRows(),
                     pokeSpamCalculator.getEvolveExtra());
 
-            if (totEvol<3) {
+            if (totEvol < 3) {
                 text = String.valueOf(pokeSpamCalculator.getTotalEvolvable());
-            } else if (totEvol>=3 && evolExtra==0) {
+            } else if (totEvol >= 3 && evolExtra == 0) {
```

Comment:
As before, better to change line 1526 to:
`} else if (totEvol >= Pokespam.HOW_MANY_POKEMON_WE_HAVE_PER_ROW && evolExtra == 0) {`